.github/workflows/build.yml

```diff
@@ -206,6 +206,18 @@ jobs:
           working-directory: tools/hangar
           install-command: npm ci --ignore-scripts
 
+      - name: Install Example Dependencies (1/2)
```

Comment:
The reason I added it here is that the E2E test job is not installing all of the dependencies in the repo, only the ones for the hangar. We could install all dependencies, but my reasoning at the time was that it would extend the build times by up to a minute so I'd keep it scoped for now.